src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -1076,14 +1076,18 @@ LedgerMaster::checkAccept(std::shared_ptr<Ledger const> const& ledger)
             // We report only if (1) we have accumulated validation messages
             // from 90% validators from the UNL, and (2) 60% of validators
             // running the rippled implementation have higher version numbers.
-            constexpr std::size_t reportingPercent = 90;
-            constexpr std::size_t cutoffPercent = 60;
-            needPrint = calculatePercent(
-                            vals.size(),
-                            app_.validators().getQuorumKeys().second.size()) >=
-                    reportingPercent &&
-                calculatePercent(higherVersionCount, rippledCount) >=
-                    cutoffPercent;
+            if (higherVersionCount > 0)
```

Comment:
I'm going to make a post-approval suggestion that you add `assert(higherVersionCount <= rippledCount)` before this line, and a comment explaining that because `higherVersionCount` is dependent on `rippledCount`, we're avoiding a potential divide by 0, AND skipping the math entirely if there's no point because there are no higher versions reporting trusted validations.

Reply:
My personal inclination would be to change the `if` to look like this:
```
if (higherVersionCount > 0 && rippledCount > 0)
```
Then I'd add one more condition to the comment: " and (3) the calculation won't cause divide-by-zero."

Reply:
@ximinez updated. I took in Scott suggested since that seems more straightforward and easier to understand.